backend/api/services/authorServices.py

```diff
@@ -22,6 +28,48 @@ def getAuthorJsonById(id):
     serializer = AuthorSerializer(a)
     return Response(serializer.data)
 
+# Checks the author table for a author with a specific URL
+def getRemoteAuthor(request, remote_author_create_func):
+    requestor_url = request.headers.get('X-Request-User')
+    #author = Author.objects.get(url=requestor_url)
+    author = Author.objects.filter(url=requestor_url)
+    if author.exists():
+        return author[0]
+    # At this point the author doesnt exist so we must create
+    else:
+        author = createRemoteAuthor(request, remote_author_create_func)
+        return author    
+
+# Create a Remote author by requesting information for a remote author and creating a corresponding author
```

Comment:
Why we need to create an `author` for remote user? I thought we should send a request and asking the external server to update `like` `comment` `post` etc in their own database